### PR TITLE
Change CairoMakie to GLMakie in Docs and examples

### PR DESCRIPTION
**Pull request by navidcy**
Use GLMakie in Docs built instead of CairoMakie.

This change fixes a visualization glitch in the Docs. Also produces a better 3D plot without the need to hardcode alpha for each panel.

@Sbozzolo, let's include this in the v0.3.0 release

### CODE REVIEW

**Comment by navidcy** [comment]
@Sbozzolo if you approve feel free to merge!

File: .github/workflows/Documenter.yml [hunk 1]
```diff
@@ -11,20 +11,26 @@ jobs:
...
     steps:
-      - name: Cancel Previous Runs
-        uses: styfle/cancel-workflow-action@0.8.0
-        with:
-          access_token: ${{ github.token }}
+      - name: Install binary dependencies
+        run: sudo apt-get update && sudo apt-get install -y xorg-dev mesa-utils xvfb libgl1 freeglut3-dev libxrandr-dev libxinerama-dev libxcursor-dev libxi-dev libxext-dev
       - uses: actions/checkout@v4
+      - name: Download all workflow run artifacts
+        uses: actions/download-artifact@v4
```
Comment (Sbozzolo) [inline .github/workflows/Documenter.yml:18]: Why is this needed?

Comment (navidcy) [reply .github/workflows/Documenter.yml:18]: not sure; shall we try delete?

Comment (Sbozzolo) [reply .github/workflows/Documenter.yml]: Yes, I don't think it should be needed

**Review by Sbozzolo — APPROVED** [review]
